lib/ColorScheme.cpp

```diff
@@ -742,12 +747,13 @@ bool ColorSchemeManager::deleteColorScheme(const QString& name)
 QString ColorSchemeManager::findColorSchemePath(const QString& name) const
 {
 //    QString path = KStandardDirs::locate("data","konsole/"+name+".colorscheme");
-    QString path(get_color_schemes_dir() + "/"+ name + ".colorscheme");
+    const QString dir = get_color_schemes_dirs().first();
+    QString path(dir + "/"+ name + ".colorscheme");
     if ( !path.isEmpty() )
```

Comment:
Here the `path` will never be empty, as it is constructed as concatenation of at least one non-empty string.